schema.sql

```diff
@@ -64,4 +64,16 @@ CREATE TABLE visits (
         REFERENCES vets(id),
     CONSTRAINT fk_animals FOREIGN KEY (animals_id)
         REFERENCES animals(id)
-);
\ No newline at end of file
+);
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
```

Comment:
[ ] [optional] @anthonymr and @zewdu444 great work on your queries you could also try adding the index on the case-insensitive version of an email